src/settings/components/Capabilities/CapabilitiesProcedural.js

```diff
@@ -24,13 +24,16 @@ const CapabilitiesProcedural = ({ content, readOnly, onChangeCapabilityCheckbox
   const resultsFormatter = {
     application: item => getApplicationName(item.applicationId),
     resource:item => item.resource,
-    execute: item => <CheckBoxWithAsterisk
-      aria-describedby="asterisk-policy-desc"
-      aria-label={getCheckBoxAriaLabel('execute', item.resource)}
-      onChange={event => onChangeCapabilityCheckbox(event, item.id, 'execute')}
-      readOnly={readOnly}
-      checked={item.actions.execute}
-    />,
+    execute: item => {
+      if (!readOnly && item.action !== 'execute') return null;
```

Comment:
Can you add a comment explaining why we don't show a checkbox given these conditions? It is not immediately obvious to me how those conditions relate to hiding it.

Reply:
Added docstrings